datafeeder/api.yaml

```diff
@@ -0,0 +1,342 @@
+openapi: 3.0.0
+servers:
+  - description: SwaggerHub API Auto Mocking
+    url: https://sample.url/import
+info:
+  title: DataFeeder API
+  description: This API covers dataset upload and publishing features of the application
+  version: "1.0.0"
+  contact:
+    email: psc@georchestra.org
+tags:
+  - name: File Upload
+    description: 'API to control jobs related to form based file upload of geospatial datasets'
+  - name: Data Publishing
+    description: 'API to control jobs related to publishing geospatial datasets previously uploaded'
+
+paths:
+  /upload:
+    post:
+      tags:
+        - File Upload
+      description: 'Upload one or multiple files to the server, and get a handle to the job'
+      operationId: uploadFiles
+      requestBody:
+        content:
+          multipart/form-data:
+            schema:
+              type: object
+              properties:
+                filename:
+                  type: array
+                  items:
+                    type: string
+                    format: binary
+      responses:
+        202:
+          $ref: '#/components/responses/UploadStatusResponse'
+        401:
+          description: 'Not authenticated'
+        403:
+          description: 'Forbidden. User has no prviledges to upload files'
+    get:
+      tags:
+        - File Upload
+      description: 'Poll the status of all the upload jobs owned by the calling user'
+      operationId: findUserUploadJobs
+      responses:
+        200:
+          $ref: '#/components/responses/UploadStatusResponseList'
+        401:
+          description: 'Not authenticated'
+        403:
+          description: 'Forbidden. User has no prviledges to query other users jobs'
+
+  /upload/all:
+    get:
+      tags:
+        - File Upload
+      description: 'Poll the status of a given upload job by id'
```

Comment:
`of a given upload job by id` confusing with `all`

Reply:
Thanks, fixed at 4fc1613